# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/AudioData.cpp
# Commit: 2f4654d81e91
# Full Hash: 2f4654d81e91156211a7cd0e53c5aef3a9728de0
# Author: Iulian Moraru <imoraru@mozilla.com>
# Date: 2024-03-07 09:55:57
# Regressor Bug: 1858958
# File Overlap Count: 1
# Description:
#   Bug 1858958 - fix build bustages. r=fix CLOSED TREE
# ==============================================================================

diff -r 2960ea3e50ca -r 2f4654d81e91 dom/media/webcodecs/AudioData.cpp
--- a/dom/media/webcodecs/AudioData.cpp	Wed Mar 06 16:32:56 2024 +0000
+++ b/dom/media/webcodecs/AudioData.cpp	Wed Mar 06 19:34:38 2024 +0200
@@ -149,6 +149,8 @@
     case AudioSampleFormat::S32_planar:
     case AudioSampleFormat::F32_planar:
       return sizeof(float);
+    default:
+      MOZ_ASSERT_UNREACHABLE("wrong enum value");
   }
 }
 
@@ -205,6 +207,8 @@
       return "s32-planar";
     case AudioSampleFormat::F32_planar:
       return "f32-planar";
+    default:
+      MOZ_ASSERT_UNREACHABLE("wrong enum value");
   }
 }
 
